R/utils.R

```diff
@@ -205,30 +205,58 @@ canonize <- function(
 ) {
 
   # args <- list(...)
+  if(!is.null(stat_label)) validate(stat_label, check_class = "character", check_length = 1L)
+  if(!is.null(est_label))  validate(est_label, check_class = "character", check_length = 1L)
 
   conf_level <- attr(x$conf.int, "conf.level")
   if(is.null(conf_level)) {
-    conf_level <- attr(x$conf.int[[1]], "conf.level")
+    conf_level <- attr(x$conf.int, "conf.level") <- attr(x$conf.int[[1]], "conf.level")
   }
   conf_label <- paste0(
     if(!is.null(conf_level)) paste0(conf_level * 100, "\\% ")
     , "CI"
   )
-  # print(conf_label)
+
+  # Corrections of df from
+  # - Hierarchical Linear Models: Kenward-Roger and Satterthwaite
+  # - Repeated-measures ANOVA   : Greenhouse-Geisser and Huyhn-Feldt
+  correction_type <- attr(x, "correction")
+  if(is.null(correction_type)) {
```

Comment:
Wouldn't it be easier to paste `^{...}` in case of a non-null `correction_type`?

Reply:
Hm, true.

Reply:
done